metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -8,118 +8,103 @@
       type: keyword
       description: >
         UUID of the Elasticsearch cluster to which Kibana connects.
-    - name: name
-      type: keyword
-      description: >
-        Kibana instance name.
-    - name: uuid
-      type: keyword
-      description: >
-        Kibana instance uuid.
-    - name: version.number
-      type: keyword
-      description: >
-        Kibana version number.
-    - name: status.overall.state
-      type: keyword
+    - name: concurrent_connections
+      type: long
       description: >
-        Kibana overall state.
+        Number of client connections made to the server. Note that browsers can send multiple simultaneous connections to request mulitple server assets at once, and they can re-use established connections.
     - name: process
       type: group
       description: >
-        Kibana process metrics.
+        Process metrics
       fields:
-        - name: mem
-          type: group
-          description: >
-            Memory usage metrics of the Kibana instance.
-          fields:
-            - name: heap.max.bytes
-              type: long
-              format: bytes
-              description: >
-                Total amount of heap memory used by V8.
-            - name: heap.used.bytes
-              type: long
-              format: bytes
-              description: >
-                Amount of memory in use by V8.
-            - name: resident_set_size.bytes
-              type: long
-              format: bytes
-              description: >
-                The amount of space occupied in main memory for the process. Includes heap, code segment, and stack.
-            - name: external.bytes
-              type: long
-              format: bytes
-              description: >
-                Memory usage of C++ objects bound to JavaScript objects managed by V8.
-        - name: pid
```

Comment:
I'm tempted to say we should keep the `pid` and put it under `process.pid` for correlation reasons.

Reply:
Added back in 332c83e.